numpy/core/_add_newdocs.py

```diff
@@ -2740,6 +2740,8 @@
 
     Toggle between low-endian and big-endian data representation by
     returning a byteswapped array, optionally swapped in-place.
+    Arrays of strings are not swapped. The real and imaginary parts
```

Comment:
This comment isn't true - bytes _are_ swapped for arrays of (unicode) strings:
```
>>> A = np.array(['ceg', 'fac'])
>>> A.view(np.uint8)
array([ 99,   0,   0,   0, 101,   0,   0,   0, 103,   0,   0,   0, 102,
         0,   0,   0,  97,   0,   0,   0,  99,   0,   0,   0], dtype=uint8)

>>> A.byteswap().view(np.uint8)
array([  0,   0,   0,  99,   0,   0,   0, 101,   0,   0,   0, 103,   0,
         0,   0, 102,   0,   0,   0,  97,   0,   0,   0,  99], dtype=uint8)

Reply:
You are right. Should we remove the comment about strings and the example below?

Reply:
The example is actually correct. It will raise `UnicodeDecodeError` when swapping arrays of strings.

Reply:
The example does not show what you think it does.
This works fine:
```
>>> A = np.array(['ceg', 'fac'])
>>> B = A.byteswap()  # no error
>>> print(B)  # UnicodeDecodeError
```

I think that example was trying to show what happens with _byte-strings_, so should have been `np.array([b'ceg', b'fac'])`

Reply:
I see. I can update the example to show what happens with byte-strings. Should I also add an example to show ``arr.newbyteorder().byteswap()`` here?